src/auro-nav.js

```diff
@@ -235,11 +235,8 @@ export class AuroNav extends LitElement {
     icon.category = category;
```

Comment:
line:232, `customSize` attribute is deprecated, we can remove this line

Reply:
fixed this